GameOfLife/Calculations/GameFieldSize.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class GameFieldSize
+    {
+        // Field size set up  2D array
+        public int SelectLenght()
+        {
+            Console.WriteLine("Enter Lenght");
+
+            try
+            {
+               int lenght = Convert.ToInt32(Console.ReadLine());
+                if(lenght <= 0)
+                {
+                    Console.WriteLine("Invalid input try again!");
+                    return SelectLenght();
+                }
+               return lenght;
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Invalid input try again!");
+                return SelectLenght();
+            }
+        
+        }
+
+        public int SelectWidth()
```

Comment:
This method uses similar logic as other for length. Consider to create generic method to retrieve value for needed parameter. Message to user should be supplied to that method, but processing and validation would be the same.